### PR TITLE
Bumped some dependencies

### PR DESCRIPTION
**Pull request by tokafish**
Just got going with a new phoenix app - and I had to make a few tweaks to use phoenix_slim with it

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -20,10 +20,10 @@ defmodule PhoenixSlim.Mixfile do
   end
 
   def deps do
-    [{:phoenix, "~> 0.15.0"},
+    [{:phoenix, "~> 0.16", only: [:dev, :test]},
```
Comment (doomspork) [inline mix.exs]: Mind if I ask why only `[:dev, :test]`?

**Comment by doomspork** [comment]
Thanks @wless1!  Only question I have is about limiting Phoenix to `[:dev, :test]`.  If you can clear that up for me this is good to go :+1:

**Comment by tokafish** [comment]
@doomspork I thought the dependency was only used in test, but upon closer look, that was a mistake. I've updated the commit to undo that.

**Comment by doomspork** [comment]
Looks good, thank you :+1:

**Pull request merged**